pytorch_blade/tests/torch-disc-pdll/utils.cpp

```diff
@@ -32,6 +34,7 @@ const std::string kDefaultHelperFunctionDeclarations = R"pdll(
   Rewrite CreateTorchCustomCall(tag : Attr, inputs : ValueRange, outputs : ValueRange) -> (op: Op, new_outputs : ValueRange);
   Rewrite ConvertTorchConstantIntListToI64DenseElemsAttr(cst: Value) -> Attr;
   Rewrite ConvertTorchConstantIntToI64Attr(cst: Value) -> Attr;
+  Rewrite ConvertTorchTensorElemType(old_type: TypeRange, type_str: Attr) -> TypeRange;
```

Comment:
Why use `TypeRange` instead of `Type`?

Reply:
`Type` is more compatible here, use it instead.